src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -319,15 +303,6 @@ WebsocketTransport.prototype.setErrorCallback = function(callback) {
     this.errorCallback = callback;
 };
 
-/**
- *
- * Suggested next step is a fallback to another transport if possible.
- * @param callback
- */
-WebsocketTransport.prototype.fail = function(callback) {

```

Comment:
Was this unused?

Reply:
yup

Reply:
Yes, we only assign it in constructor and we don't use this method, so fine to remove it.